core/src/backends/cln/grpc/pb.rs

```diff
@@ -105,3 +108,39 @@ impl Into<PayInvoiceResult> for PayResponse {
         }
     }
 }
+
+impl TryInto<DecodeInvoiceResult> for String {
+    type Error = Error;
+
+    fn try_into(self) -> Result<DecodeInvoiceResult, Error> {
+        // DecodeInvoice gRPC command is not implemented yet on CLN, so we need to use an external parser instead
+        let parsed_invoice = str::parse::<lightning_invoice::Invoice>(self.as_ref())
+            .expect("provided invoice is not a valid bolt11 invoice");
+
+        let memo = match parsed_invoice.description() {
+            InvoiceDescription::Direct(direct) => Some(direct.to_string()),
+            InvoiceDescription::Hash(_hash) => {
+                unimplemented!("Hash transcription is not supported yet")
```

Comment:
```suggestion
            lightning_invoice::InvoiceDescription::Direct(direct) => Some(direct.to_string()),
            lightning_invoice::InvoiceDescription::Hash(_hash) => {
                unimplemented!("Hash transcription is not supported yet")
```
That way we can see directly that we are using an external crate (also to keep consistency with the `lightning_invoice::Invoice` above).